sdk/storagesync/Azure.ResourceManager.StorageSync/tests/Tests/CloudEndpointTests.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.ResourceManager.StorageSync.Models;
+using System.Collections.Generic;
+using NUnit.Framework;
+using System.Linq;
+using Azure.Core.TestFramework;
+using System.Threading.Tasks;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.StorageSync.Tests
+{
+    public class CloudEndpointTests : StorageSyncManagementTestBase
+    {
+        private ResourceGroupResource _resourceGroup;
+
+        private string _storageSyncServiceName;
+        private string _syncGroupName;
+        private string _cloudEndpointName;
+        private StorageSyncServiceCreateOrUpdateContent _storageSyncServiceCreateOrUpdateContent;
+        private StorageSyncGroupCreateOrUpdateContent _storageSyncGroupCreateOrUpdateContent;
+        private CloudEndpointCreateOrUpdateContent _cloudEndpointCreateOrUpdateContent;
+        private StorageSyncServiceResource _storageSyncServiceResource;
+        private StorageSyncGroupResource _storageSyncGroupResource;
+
+        public CloudEndpointTests(bool async) : base(async) //, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public async Task CreateStorageSyncResources()
+        {
+            _resourceGroup = await CreateResourceGroupAsync();
+            _storageSyncServiceName = Recording.GenerateAssetName("sss-cepcreate");
+            _syncGroupName = Recording.GenerateAssetName("sg-cepcreate");
+            _cloudEndpointName = Recording.GenerateAssetName("cepcreate");
+
+            _storageSyncServiceCreateOrUpdateContent = StorageSyncManagementTestUtilities.GetDefaultStorageSyncServiceParameters();
```

Comment:
Why are they not abstracted out in common test class setup - whatever make sense